### DISCUSSION TITLE
postgresql wal 에 쓰기는 mmap 방식으로 처리될까 I/O 방식으로 처리될까

### DISCUSSION THREAD

**Discussion by JONGWOONG-SEON** [kyungjunleeme/kafka_study#47] [Q&A]
https://github.com/facebook/rocksdb/issues/11835

둘 다 직접적으로 구현 가능함

**Comment by kyungjunleeme** [comment]
덕분에 postgresql 관련해서도 조금 알게되었습니다. 감사합니다.

**Comment by kyungjunleeme** [comment]
cf) 파이썬도 mmap이 있군요
https://docs.python.org/ko/3.12/library/mmap.html
